assembly/src/main/dist/docker/build.sh

```diff
@@ -30,4 +30,4 @@ cp $ZIP ./ignore/rdf4j.zip
 # build
 docker-compose build
 
-docker tag docker_rdf4j:latest eclipse/rdf4j-workbench:${MVN_VERSION}
\ No newline at end of file
+docker tag docker_rdf4j:latest eclipse/rdf4j-workbench:${MVN_VERSION}
```

Comment:
I'm a little lost on what this script produces. Clearly, the end result of this will be a locally tagged docker image using just the version, so, e.g., `eclipse/rdf4j-workbench:3.4.4`. So where and how do we produce the different images for the ARM and AMD architectures, and how are they pushed to dockerhub?

Reply:
FWIW, currently I push the dockers manually to dockerhub. 
There *might* (haven't checked) be some github action that could build/push (with some secret key file) for different architectures.

Reply:
@barthanssens can you talk me through how you currently produce the docker build, step by step? Perhaps on the dev list, rather than here.